docs/developer/README.md

```diff
@@ -84,6 +84,7 @@ The endpoint /admin can be used to view and manage site content, including but n
 ```
 
 5. In the browser, navigate to /admin. To verify that all is working correctly, under "domain applications" you should see fake domains with various fake statuses.
+6. Add an optional email key/value pair
```

Comment:
My only suggestion here would be to include an example somewhere, eg: "email" = "<Optional Email Here>"

Reply:
I think the context is strong enough (a few lines before the diff, there's a code block with example firnatname, lastanme, etc.)